SmartDeviceLink/public/SDLLockScreenViewController.m

```diff
@@ -61,6 +62,10 @@ - (UIStatusBarStyle)preferredStatusBarStyle {
     return useWhiteIcon ? UIStatusBarStyleLightContent : UIStatusBarStyleDefault;
 }
 
+- (UIModalPresentationStyle)modalPresentationStyle {
+    return _modalPresentationStyle;
+}
```

Comment:
This is not correct. You are not trying to re-define it, you're trying to override it.

```suggestion
- (UIModalPresentationStyle)modalPresentationStyle {
    return UIModalPresentationFullScreen;
}
```